go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -609,6 +609,12 @@ func (erp *EmergencyReparenter) reparentReplicas(
 			// we're going to be explicit that this is doubly unexpected.
 			return nil, vterrors.Wrapf(rec.Error(), "received more errors (= %d) than replicas (= %d), which should be impossible: %v", errCount, numReplicas, rec.Error())
 		case errCount == numReplicas:
+			if len(tabletMap) <= 2 {
+				// If there are atmost 2 tablets in the tablet map, we shouldn't be failing the promotion if the replica fails to SetReplicationSource.
```

Comment:
Two minor typos:
```
// If there are at most 2 tablets in the tablet map, we shouldn't be failing the promotion if the replica fails to SetReplicationSource.
// The failing replica is probably the old primary that is down, so it is okay if it fails. We still log a warning message in the logs.
```

I'll leave it up to you if it's worth another CI run 🙂

Reply:
Fixed.